src/themes/mainTheme.ts

```diff
@@ -166,3 +166,8 @@ export const autocompleteTranslationProps = {
   closeText: i18n.t('common:close'),
   openText: i18n.t('common:open'),
 };
+
```

Comment:
bare lurer på hvorfor translation-props legges i theme ?

Reply:
LInje 152-156 setter de som default props siden komponentenen kan brukes flere plasser er det enklere å bare legge de der, siden oversettelsene skal være like overalt. alternativet ville vært å sende de som ny prop alle plasser komponenten brukes, som fort blir styr hvis man skal endre noe og TS ikke kan vite at en i18n key har blitt endret.

Men det kan bare settes direkte i theme om komponenten er i MUI/core, så for de andre legger vi de bare under her (så kan de forhåpentligvis flyttes opp som default props de også når de omsider flyttes til MUI/core)